gradle-plugin/src/main/groovy/io/realm/gradle/RealmRobolectric.groovy

```diff
@@ -0,0 +1,35 @@
+package io.realm.gradle
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+
+class RealmRobolectric implements Plugin<Project> {
+
+    @Override
+    void apply(Project project) {
+        project.task('downloadRealmRobolectric') {
+            doLast {
+                def targetZip = "${project.projectDir}/realm-robolectric-darwin-${Version.VERSION}.zip"
+                def targetLib = "${project.projectDir}/src/test/libs/librealm-jni-darwin.dylib"
+
+                if (project.file(targetZip).exists() && project.file(targetLib).exists()) {
+                    println "librealm-jni-darwin.dylib is already exsit. Skip download the Realm Robolectric."
```

Comment:
The task should define inputs and outputs so that Gradle can figure out by itself whether the task is `UP-TO-DATE` or not.